liquibase-standard/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -515,14 +519,14 @@ protected Object readDefaultValue(CachedRow columnMetadataResultSet, Column colu
             Object defaultValue = columnMetadataResultSet.get(COLUMN_DEF_COL);
 
             if (("(NULL)".equals(defaultValue))) {
-                columnMetadataResultSet.set(COLUMN_DEF_COL, new DatabaseFunction("null"));
+                columnMetadataResultSet.set(COLUMN_DEF_COL, new DatabaseFunction(NULL_VALUE));
             }
         }
 
         if ((database instanceof OracleDatabase) && (columnMetadataResultSet.get(COLUMN_DEF_COL) == null)) {
             columnMetadataResultSet.set(COLUMN_DEF_COL, columnMetadataResultSet.get("DATA_DEFAULT"));
 
-            if ((columnMetadataResultSet.get(COLUMN_DEF_COL) != null) && "NULL".equalsIgnoreCase((String)
+            if ((columnMetadataResultSet.get(COLUMN_DEF_COL) != null) && NULL_VALUE.equalsIgnoreCase((String)
```

Comment:
I think it would look a bit better if we replace this `NULL_VALUE` check with something like:
```
StringUtil.equalsWordNull((String)
                    columnMetadataResultSet.get(COLUMN_DEF_COL))
```

Reply:
O! I see. I will fix it.

Reply:
Added